validator_client/validator_services/src/attestation_service.rs

```diff
@@ -709,6 +714,38 @@ impl<S: ValidatorStore + 'static, T: SlotClock + 'static> AttestationService<S,
     }
 }
 
+async fn poll_head_event_on_all_beacon_nodes<T: SlotClock, E: EthSpec>(
+    beacon_nodes: &Arc<BeaconNodeFallback<T>>,
+) -> Result<(), String> {
+    beacon_nodes
+        .first_success(|beacon_node| async move {
+            let mut event_stream = beacon_node
+                .get_events::<E>(&[EventTopic::Head])
+                .await
+                .map_err(|e| format!("Failed to get event stream: {:?}", e))?;
+
+            // Poll once for a head event to trigger early attestation processing
+            if let Some(event_result) = event_stream.next().await {
+                let event =
+                    event_result.map_err(|e| format!("Head event stream error: {:?}", e))?;
+                match event {
+                    EventKind::Head(_) => {
+                        trace!("Received head event, triggering early attestation processing");
```

Comment:
This could probably be a debug log (we almost never show trace logs), and could include the slot and block root from the event, as well as something identifying the BN